keyboards/launchpad/config.h

```diff
@@ -43,7 +43,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 /* Mechanical locking support. Use KC_LCAP, KC_LNUM or KC_LSCR instead in keymap */
 #define LOCKING_SUPPORT_ENABLE
 /* Locking resynchronize hack */
-#define LOCKING_RESYNC_ENABLE
+#define LOCKING_RESYNC_ENABLE =
```

Comment:
Not sure if the changes to this block are intentional -  the random '=' looks suspect.

Reply:
reverted file

Reply:
Looks like this has snuck back.